src/soundfile.js

```diff
@@ -902,6 +902,15 @@ class SoundFile {
     var newPlaybackRate = midiToFreq(num) / midiToFreq(60);
     this.rate(newPlaybackRate);
   }
+
+  /**
+   * Returns the current pitch of a sound file in the MIDI note.
```

Comment:
Looks good, just a minor grammar correction: instead of "in the MIDI note" I think it should read "as a MIDI note".

In addition to this line, similar to the documentation for setPitch, maybe we should mention that the default note is assumed to be 60?